Bottom: fd4845459ea4dbfd72a502ba7df3b433ee6b42bb
Top:    b797fa4d2ec8b7c7c328ae61a9d72c01a97c5694
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-11-22 01:28:38 +0000

Refresh of test

---

diff --git a/src/draw.c b/src/draw.c
index d4afb9d..a5c3ef6 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -91,7 +91,7 @@ set_object_color (AnyObjectType *obj,
 {
   char *color;
 
-       if (warn_color     != NULL && TEST_FLAG (WARNFLAG,     obj)) color = warn_color;
+  if      (warn_color     != NULL && TEST_FLAG (WARNFLAG,     obj)) color = warn_color;
   else if (selected_color != NULL && TEST_FLAG (SELECTEDFLAG, obj)) color = selected_color;
   else if (found_color    != NULL && TEST_FLAG (FOUNDFLAG,    obj)) color = found_color;
   else                                                              color = normal_color;
